### ISSUE TITLE
Click Show Hidden Item and it will be automatically set to hidden because of the TOTP countdown.

### ISSUE THREAD

**Issue by likevi9528** [bitwarden/android#3858]
### Bitwarden Beta

- [X] I'm using the new native Bitwarden Beta app and I'm aware that legacy .NET app bugs should be reported in [bitwarden/mobile](https://github.com/bitwarden/mobile)

### Steps To Reproduce

1. Set password or Hidden Fields
2. Set TOTP
3. Save
4. Click ' 👁️ ' to show password

### Expected Result

Click ' 👁️ '  to display hidden items ~~or passwords~~ all the time.

### Actual Result

Hidden items are automatically set to hidden after being displayed for one second or less.

### Screenshots or Videos

In the 2024.9.0(19104) version, the password will not be automatically hidden due to the TOTP countdown, but the Hidden item set in Custom fields will still be automatically hidden due to the TOTP countdown.

**I also report it at #3405**

https://github.com/bitwarden/android/assets/81839775/d6cb9ebb-bf1b-4e11-85ad-d0bcfd73c38c

### Build Version

2024.9.0

### Environment Details

- Device: Redmi k30
- OS: Android 12

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by cbbit** [comment]
Hi,

Thank you for your report! It has been escalated for further investigation. If you have more information that can help us, please add it below.

Thanks!

**Comment by maxmini1** [comment]
Hi @likevi9528 : this has been fixed on Version: 2024.11.3 (19432)
Try close this.

https://github.com/user-attachments/assets/77ab38d5-c23b-4692-8061-6948cd856575

**Comment by likevi9528** [comment]
> Hi @likevi9528 : this has been fixed on Version: https://github.com/bitwarden/android/releases/tag/v2024.11.3
> 
>  ScreenRecording_20241109_002836.mp4

No no no, please add a totp code and it will work as shown in this [video](https://github.com/bitwarden/android/assets/81839775/d6cb9ebb-bf1b-4e11-85ad-d0bcfd73c38c). As of latest actions["Build #8444"](https://github.com/bitwarden/android/actions/runs/11751615061), this bug still exists.

**Comment by maxmini1** [comment]
@likevi9528
You're right, that's my fault, I didn't pay attention to the second step. "Set TOTP" I apologize

**Comment by LukeWheeldon** [comment]
fwiw I am experiencing the same issue here on latest version of GrapheneOS.

**Comment by closebot-bw** [comment]
⚠️ **Stale Issue Notice**

This issue has been automatically marked as stale due to inactivity. It will be closed in **2 weeks** (December 02, 2025) if no further activity occurs.

If this issue is still relevant and you would like to keep it open, please:
- Comment on this issue to show continued interest
- Provide any additional information or updates
- Confirm that the issue still exists in the latest version

Thank you for your contribution to this project! 🙏

**Comment by closebot-bw** [comment]
🔔 **Final Notice - Issue Will Be Closed Soon**

This issue was previously marked as stale and will be automatically closed in **3 days** (December 02, 2025) if no further activity occurs.

If you're still experiencing this issue or believe it should remain open, please comment below to prevent automatic closure.

We appreciate your understanding and contribution to keeping our issue tracker organized! 📋

**Comment by closebot-bw** [comment]
🔒 **Issue Closed Due to Inactivity**

This issue has been automatically closed due to lack of activity for an extended period. We periodically review and close inactive issues to help maintain our issue tracker and focus on current priorities.

**If this issue is still relevant:**
- Please create a new issue with updated information
- Include steps to reproduce the problem if it's a bug report
- Mention if this issue still occurs in the latest version

Thank you for your contribution to this project. Your feedback helps us improve! 🙏
